### PR TITLE
Added properties for several AWS resources

### PR DESCRIPTION
**Pull request by nwmqpa**

### CODE REVIEW

File: resources/cloudwatchevents-targets.go [hunk 1]
```diff
@@ -37,6 +38,10 @@ func ListCloudWatchEventsTargets(sess *session.Session) ([]Resource, error) {
 				return nil, err
 			}
 			for _, target := range targetResp.Targets {
+				if err != nil {
+					return nil, err
+				}
+
```
Comment (der-eismann) [inline resources/cloudwatchevents-targets.go:44]:
```suggestion
```

I think this was a copy-paste error

File: resources/iam-saml-provider.go [hunk 2]
```diff
@@ -45,6 +46,11 @@ func (e *IAMSAMLProvider) Remove() error {
 	return nil
 }
 
+func (e *IAMSAMLProvider) Properties() types.Properties {
+	properties := types.NewProperties()
+	return properties
+}
+
```
Comment (der-eismann) [inline resources/iam-saml-provider.go:53]: This should have one property at least

File: resources/sfn-statemachines.go [hunk 3]
```diff
@@ -76,6 +89,17 @@ func (f *SFNStateMachine) Remove() error {
 	return err
 }
 
+func (s *SFNStateMachine) Properties() types.Properties {
+	properties := types.NewProperties()
+
+	properties.Set("Name", s.name)
```
Comment (der-eismann) [inline resources/sfn-statemachines.go:95]: This should have the ARN as well

**Review by der-eismann — COMMENTED** [review]
Hey @nwmqpa! Sorry for the late reply. Are you still interested in getting your changes in?  It would be great if you could implement my suggestions then.

**Comment by nwmqpa** [comment]
Hey @der-eismann, I'll try to implement those soon

**Comment by ekristen** [comment]
@nwmqpa due to the size and the number of resources touched here and that the fork as since diverged a bit, I cannot pull these over cleanly. If you'd like to open a PR on https://github.com/ekristen/aws-nuke it would be welcome, please read the contributing guide over there.

Otherwise, It's on my backlog to add properties to all resources. Thanks.

----

Please see the copy of the notice from the README about the deprecation of this project. Sven was kind enough to grant me access to help triage and close issues and pull requests that have already been addressed in the actively maintained fork. Some additional information is located in the [welcome issue](https://github.com/ekristen/aws-nuke/issues/276) for more information. 

> [!CAUTION]
> This repository for aws-nuke is no longer being actively maintained. We recommend users to switch to the actively maintained fork of this project at [ekristen/aws-nuke](https://github.com/ekristen/aws-nuke).
> We appreciate all the support and contributions we've received throughout the life of this project. We believe that the fork will continue to provide the functionality and support that you have come to expect from aws-nuke.
> Please note that this deprecation means we will not be addressing issues, accepting pull requests, or making future releases from this repository.
> Thank you for your understanding and support.

**Pull request closed without merging**